frontend/workflows/projectSelector/src/selector-reducer.tsx

```diff
@@ -280,6 +279,9 @@ const selectorReducer = (state: State, action: Action): State => {
         error: action?.payload?.result,
       };
     }
+    case "CLEAR_ERRORS": {
```

Comment:
It's a bit unclear by the name what errors are being cleared (e.g. it looks like there are project errors vs a global error)

Reply:
Good call, renamed this event to `CLEAR_PROJECT_ERRORS` to make it clear what is happening.